uber/model_checks.py

```diff
@@ -24,6 +24,13 @@ def duplicate_admin(account):
             if session.query(AdminAccount).filter_by(attendee_id=account.attendee_id).all():
                 return 'That attendee already has an admin account'
 
+@validation.AdminAccount
+def duplicate_admin(account):
+    if account.is_new:
+        with Session() as session:
+            if session.query(Attendee).filter_by(id=account.attendee_id).first().email == '':
```

Comment:
There's a convenience method you could use here; you can just say ``session.attendee(account.attendee_id)`` to get the Attendee with that id.  This exists for all of our models (e.g. you could say ``session.job(some_id)`` or ``session.event(another_id)``).

So in this case you should probably rewrite this if statement to just say

```python
if not session.attendee(account.attendee_id).email:
```

(In this case I did ``if not`` instead of ``== ''`` but that's mostly a stylistic preference, and you could do it either way.)